test/dotnet-publish.Tests/PublishAppWithBuildDependency.cs

```diff
@@ -17,8 +17,6 @@ public void PublishExcludesBuildDependencies()
             var testInstance = TestAssetsManager.CreateTestInstance("AppWithDirectDependencyAndTypeBuild")
                 .WithLockFiles();
 
-            new RestoreCommand() { WorkingDirectory = testInstance.TestRoot }.Execute().Should().Pass();
```

Comment:
A restore is already happening in the build scripts, so this is just an extra restore without any custom args.